edge-util/src/Microsoft.Azure.Devices.Edge.Storage/BackupRestoreUtil.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is this needed? It seesm that IBackupRestore implementation could have a property for the SerializationFormat

Reply:
Good point, changed IBackupRestore or rather IDataBackupRestore (new name) to have a format property that returns this instead.